chips/nrf52/src/ppi.rs

```diff
@@ -0,0 +1,90 @@
+use kernel::common::VolatileCell;
+
+pub const PPI_BASE: usize = 0x4001F000;
+#[repr(C)]
+pub struct PPIRegs {
```

Comment:
We should use the new register interface for this. See `chips/nrf52/src/clock.rs` for an example